CHANGES.rst

```diff
@@ -6,6 +6,8 @@ v3.0.4
     * A `garden.yaml` file was added for use with the
       `garden <https://crates.io/crates/garden-tools>_` command runner. (+486)
     * The test suite was updated to avoid deprecated SQLALchemy APIs.
+    * `jsonpickle.ext.numpy.register_handlers` now provides options that are forwarded
```

Comment:
This is definitely an API addition, so this should be v3.1.0.

Reply:
Excellent. I'll go ahead and start prepping the v3.0.4 release before merging this in.